### PR TITLE
docs: document arguments

### PR DESCRIPTION
**Pull request by ysndr**

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -1,19 +1,102 @@
...
   # =================================
 
+  outputs = args @ {flox-floxpkgs, ...}:
+    flox-floxpkgs.project args (context: {
+      # explicit package definitions
+      # the format is equivalent to package definitions in
+      # `./pkgs/my-pkg/default.nix`
+      packages = {
+        # explicit packages allow re-exports of single packages
+        # from package sets that can't be accessed by all users of this project
```
Comment (jurraca) [inline flake.nix:22]: when would this be the case? (users not being able to access a package in the project)

File: flake.nix [hunk 2]
```diff
@@ -1,19 +1,102 @@
...
+        #
+        # Note: re-exporting generally provides less guarantees
+        #       than importing a package set in terms of composability.
```
Comment (jurraca) [inline flake.nix:25]: could you add a little bit about what composability guarantees can be lost here?

**Review by jurraca — APPROVED** [review]
looks great, a few things to elaborate on if you think it's relevant

**Comment by stahnma** [comment]
What needs to happen to get this across the line? This appears to have sat idle for 90 days.

**Comment by ysndr** [comment]
@stahnma I think this just got lost
I'ts basically just adding a documentation so rebasing should be fairly easy

**Pull request merged**